test/lib/import/colorectal/providers/st_george/st_george_handler_colorectal_test.rb

```diff
@@ -0,0 +1,376 @@
+require 'test_helper'
+
+class StGeorgeTest < ActiveSupport::TestCase
```

Comment:
Could the class name match the file name please? e.g. `StGeorgeHandlerColorectalTest`

Reply:
ammended in [075175e](https://github.com/NHSDigital/data_management_system/commit/075175e335e5a81d699eebf4501ad28f9d5a8048)